docs/contribute-docs.md

```diff
@@ -61,16 +82,11 @@ Here's a brief overview of where to look for things:
   - [Book theme source](https://github.com/jupyter-book/myst-theme/tree/main/themes/book) -> [Book theme built version](https://github.com/myst-templates/book-theme)
   - [Article theme source](https://github.com/jupyter-book/myst-theme/tree/main/themes/article) -> [Article theme built version](https://github.com/myst-templates/article-theme)
 
-### Modifying and releasing a theme
-
-Below is a brief description for how modifications in a theme are released for public consumption.
+The **content of mystmd.org** is pulled from a number of repositories across the `jupyter-book` organization.
 
-1. **[dev facing]** we make a bunch of changes to the theme, probably changing things in `themes/book/*`, `themes/article/*` and `packages/*`
-2. **[dev facing]** if there were changes in `packages/*` we release the `@myst-theme/*` packages (changesets based ci step)
-3. **[dev facing]** we then run `make deploy-book` and `make deploy-article` which builds and bundles each theme, making commits to the https://github.com/myst-templates/book-theme and https://github.com/myst-templates/article-theme repos.
-4. **[user facing]** at that point those latest bundle commits will be what is pulled by `mystmd` clients - people who already have those downloaded need to `myst clean --all` (as we don't yet auto bump based on version changes, see [#854](https://github.com/jupyter-book/mystmd/issues/854) for updates).
+For example, `mystmd.org/guide` is pulled from [`jupyter-book/mystmd: /docs/`](https://github.com/jupyter-book/mystmd/tree/main/docs), and [`mystmd.org/jtex`](https://mystmd.org/jtex) is pulled from [`jupyter-book/mystmd: /jtex/docs/`](https://github.com/jupyter-book/mystmd/tree/main/packages/jtex/docs). It also includes some custom applications like the Sandbox and MyST demo on the landing page.
```

Comment:
Good to have these year; a full list would be helpful, but annoying to duplicate. Could we link to the Vercel config, or commit that config to the repo like we do with Netlify?

Reply:
I have never used Vercel, I'm afraid!

Reply:
There is no config. Just select "remix" preset from the dropdown. We aren't doing anything fancy here!